soco/core.py

```diff
@@ -64,16 +64,19 @@
     18: "Dolby 5.1",
     21: "No input",
     22: "No audio",
-    59: "Dolby Atmos",
-    63: "Dolby Atmos",
+    59: "Dolby Atmos (DD+)",
+    61: "Dolby Atmos (TrueHD)",
+    63: "Dolby Atmos (MAT 2.0)",
     33554434: "PCM 2.0",
     33554454: "PCM 2.0 no audio",
     33554488: "Dolby 2.0",
     33554490: "Dolby Digital Plus 2.0",
+    33554492: "Dolby TrueHD 2.0",
     33554494: "Dolby Multichannel PCM 2.0",
     84934658: "Multichannel PCM 5.1",
     84934713: "Dolby 5.1",
     84934714: "Dolby Digital Plus 5.1",
+    84934721: "DTS 5.1"
```

Comment:
Very minor style nitpick.
```suggestion
    84934721: "DTS 5.1",
```